tests/apis.js

```diff
@@ -0,0 +1,430 @@
+import test from 'ava';
+import uuid from 'uuid';
+import ipsum from 'lorem-ipsum';
+import Promise from 'bluebird';
+import slugify from 'underscore.string/slugify';
```

Comment:
why the switch from `lodash/kebabCase`? Now we have two different modules doing the same thing